pkgs/development/python-modules/transformers/default.nix

```diff
@@ -44,7 +44,7 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace setup.py \
-      --replace "tokenizers == 0.8.0-rc4" "tokenizers>=0.8,<0.9"
+      --replace "tokenizers == 0.8.1.rc1" "tokenizers>=0.8,<0.9"
```

Comment:
```suggestion
      --replace "tokenizers == 0.8.1.rc1" "tokenizers"
```
Would unlocking tokenizers altogether make sense?

Reply:
if we have tests, this is probably best

Reply:
Most tests for tokenizers are disabled, since they download models. But they claim some backwards compatibility. So I think we could do `>=0.8`, so with a lower bound and without an upper bound and see how that goes.

Reply:
PR updated with this bound.